src/lib/utils/thorchain/index.ts

```diff
@@ -191,7 +190,7 @@ export const getThorchainFromAddress = async ({
       accountNumber: bip44Params.accountNumber,
       accountType,
       index: 0,
-      pubKey: isLedger(wallet) && accountId ? fromAccountId(accountId).account : undefined,
```

Comment:
revert me

Reply:
We definitely want to keep this guy, see https://github.com/shapeshift/web/pull/7592#discussion_r1729078514
Else `adapter.getAddress()` would throw in case the app isn't opened/Ledger got disconnected  and `poolAssetAccountAddress` would be undefined there

Reply:
I am saying we have always been using the address on `accountId` for ledger only, and fetching for other wallets. Seems like we can/should keep it that way for address derivation needs specifically

Reply:
As per dms, best to use wallet for deriving address here if non ledger and revert the associated comment back to `fromAccountId` for gas things

Reply:
0c1d9bc540